checker/examples/Makefile

```diff
@@ -4,7 +4,6 @@ all: compile
 
 compile:
 	$(JAVAC) *.java
-	$(JAVAC) units-extension/*.java
```

Comment:
Rather than deleting this line, should it delegate to the Makefile in the units-extension/ directory?